ci/install-spellcheck.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
have we set the executable bit on this file?

Reply:
Good catch. Fixed in 969810341fd4c0095899bae6b2d05812098a0ee6.